R/nhs_footer.R

```diff
@@ -17,7 +30,7 @@ nhs_footer <- function() {
               a(
                 class = "nhsuk-footer__list-item-link",
                 style = "text-decoration: underline;",
-                href = "#",
+                href = a11y_statement_url,
```

Comment:
waht is this?

Reply:
This is the link to the accessibility statement that will be on NHSBSA website.